web/packages/demo/www/index.js

```diff
@@ -111,6 +135,11 @@ main.addEventListener("drop", (event) => {
     localFileInput.files = event.dataTransfer.files;
     loadFile(event.dataTransfer.files[0]);
 });
+main.addEventListener("click", (event) => {
```

Comment:
Now when seen in action, it feels quite annoying because I mistakenly clicked on that area too many times.

Reply:
I switched to only executing this when you have not already selected a file, which was probably the best idea in the first place.